docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,37 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Account 
+
+### Balance
+
+Used to lookup address asset balances
+
+`GET /v1/account/{address}/balance`
+
+#### Path parameters
+
+* `address` - is the address for lookup. eg. `0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
+
+#### Query parameters
+
+* `currency` - currency to calculate the assets value. e.g. `USD`
+* `chain` - (Optional) filter assets by CAIP-10 chain id.
+
+#### Success response body:
+
+* `positions` - list of address positions with the following attributes:
```

Comment:
Assuming we are just forwarding to another API, but this API would return _all_ positions which could be unnecessary and large, and would also not support a custom token ID/address

Reply:
Makes sense to me to change it to `balances`. It's changed. Thanks for the suggestion!